cves/CVE-2017-16840.yml

```diff
@@ -64,7 +64,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  When processing video media, non-Haar wavelets are used to transform 
+  the pixels in the frame. In order to do so, non-Haar wavelets need to look 
+  outside of the frame. This could cause a boundary condition error and a denial 
+  of service. The current functionality only pads the bottom and left part of the video 
+  frame. The solution makes the padding on all four sides of the frame using the 
+  slice width and height.
```

Comment:
After checking the commit message for this commit, this looks like a poor paraphrasing of that. Please try to write this in your own words.